apps/web/cypress/tests/integrations-list-page.spec.ts

```diff
@@ -32,23 +32,56 @@ describe('Integrations List Page', function () {
     }: {
       name: string;
       isFree?: boolean;
-      key: string;
+      key: string | null;
       provider: string;
       channel: string;
-      environment: string;
+      environment: string | null;
       status: string;
     },
     nth: number
   ) => {
-    cy.getByTestId('integration-name-cell').eq(nth).should('be.visible').contains(name);
-    // cy.getByTestId('integration-name-cell').eq(nth).should('be.visible').contains(key);
+    cy.getByTestId('integrations-list-table').as('integrations-table');
+    cy.get('@integrations-table')
+      .get('tr')
+      .eq(nth)
+      .getByTestId('integration-name-cell')
+      .should('be.visible')
+      .contains(name);
+    // cy.get('@integrations-table').get('tr').eq(nth).getByTestId('integration-name-cell').should('be.visible').contains(key);
```

Comment:
🧹

Reply:
it was left for the future, but I can remove it